Makefile

```diff
@@ -1,7 +1,7 @@
 GO ?= go
 GOFMT ?= gofmt
 V ?=
-PKGS = $(shell go list ./... | grep -v vendor)
+PKGS = $(shell go list ./... | grep -v vendor | grep -v test)
```

Comment:
This has been [fixed and just merged](https://github.com/mendersoftware/mender/commit/51aea56a59c9698eb0639dec3cbacf454a4faa15) a couple of minutes ago, so you can drop this patch.

Reply:
:+1: